packages/story-editor/src/app/media/utils/useFFmpeg.js

```diff
@@ -297,10 +298,11 @@ function useFFmpeg() {
       try {
         ffmpeg = await getFFmpegInstance(file);
 
-        // @todo: ensure that this function return the same file type submitted. https://github.com/google/web-stories-wp/issues/8998.
-        const tempFileName = uuidv4() + '.' + MEDIA_TRANSCODED_FILE_TYPE;
-        const outputFileName =
-          getFileName(file) + '-trimmed.' + MEDIA_TRANSCODED_FILE_TYPE;
+        const ext =
+          fileExt || getTypeFromMime(file?.type) || MEDIA_TRANSCODED_FILE_TYPE;
```

Comment:
So since `getTypeFromMime` returns an _element_ type instead of a _file_ type, we need a different solution.

Instead of a hardcoded list of mime types or a switch statement, could you explore using [`mime/lite`](https://www.npmjs.com/package/mime)? That seems like the most robust yet lightweight solution.

When we can simply do:

```js
import mime from 'mime/lite';

const fileExt = mime. getExtension(file.type)
```